src/go/rpk/pkg/cli/transform/transform.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2020 Redpanda Data, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package transform
+
+import (
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/config"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+func NewCommand(fs afero.Fs, p *config.Params) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:        "transform",
+		Aliases:    []string{"wasm"},
+		SuggestFor: []string{"transfrom"}, //nolint:misspell // this is a suggestion if a user misspells something
```

Comment:
```suggestion
		SuggestFor: []string{"transfrom"}, //nolint:misspell // this is a suggestion if a user misspells something
```

Why not just make it an alias?

Reply:
TIL `SuggestFor` nice! 

Tbf, we don't have a single use case nor a guide on what to include in suggestions vs aliases